src/main/resources/db/changelog/changelog.sql

```diff
@@ -753,3 +753,8 @@ ADD COLUMN `logoGuid` varchar(255);
 
 ALTER TABLE `Substudies`
 MODIFY COLUMN `phase` enum('LEGACY', 'DESIGN', 'RECRUITMENT', 'IN_FLIGHT', 'ANALYSIS', 'COMPLETED', 'WITHDRAWN') DEFAULT 'LEGACY';
+
+-- changeset bridge:42
+
+ALTER TABLE `Accounts`
+ADD COLUMN `clientTimeZone` varchar(64) DEFAULT NULL;
```

Comment:
Everywhere else this is 255... which might have just been done on autopilot. Does 64 cover all possible values?

Reply:
I think 64 is safe. The longest zone I could find was 32 characters (America/Argentina/ComodRivadavia).